src/providers/index.ts

```diff
@@ -1,5 +1,6 @@
 import { ServiceConfig } from '../types/general'
 
 export default {
-  dns_gd: require('./goDaddy')
+  dns_gd: require('./goDaddy'),
+  dns_namecom: require('./nameDotCom')
```

Comment:
we can do a `fs.readdir` here and make thhis generic so that new api providers don't have to set this up.